cmd/cmd.go

```diff
@@ -100,7 +99,9 @@ func CreateHandler(cmd *cobra.Command, args []string) error {
 	spinner := progress.NewSpinner(status)
 	p.Add(status, spinner)
 
-	req, err := modelfile.CreateRequest()
+	modelfileDir := filepath.Dir(filename)
+
+	req, err := modelfile.CreateRequest(modelfileDir)
```

Comment:
nit: could probably inline this `modelfile.CreateRequest(filepath.Dir(filename)))`